### PR TITLE
merge feature/anakin-engine to develop

### PR DESCRIPTION
**Pull request by Shixiaowei02**

### CODE REVIEW

File: paddle/fluid/inference/analysis/ir_passes/tensorrt_subgraph_pass.cc [hunk 1]
```diff
@@ -281,6 +280,99 @@ void TensorRtSubgraphPass::CreateTensorRTOp(
           trt_engine_serialized_data);
 }
 
+void RenameAndGetOutputs(
```
Comment (NHZlX) [inline paddle/fluid/inference/analysis/ir_passes/tensorrt_subgraph_pass.cc]: Refined here: https://github.com/PaddlePaddle/Paddle/blob/eb83abeac3c0146b921ce72d06fef2551ab3e8d8/paddle/fluid/inference/analysis/ir_passes/subgraph_util.h#L38

File: paddle/fluid/operators/conv_fusion_op.cu.cc [hunk 2]
```diff
@@ -100,6 +100,7 @@ class CUDNNConvFusionOpKernel : public framework::OpKernel<T> {
       int64_t max_user_size =
           std::max(static_cast<int64_t>(FLAGS_conv_workspace_size_limit),
                    user_workspace_size);
+      max_user_size = 64;
```
Comment (NHZlX) [inline paddle/fluid/operators/conv_fusion_op.cu.cc]: hard code here

File: paddle/fluid/inference/analysis/ir_passes/tensorrt_subgraph_pass.cc (hunk 1 above)
Comment (Shixiaowei02) [reply paddle/fluid/inference/analysis/ir_passes/tensorrt_subgraph_pass.cc]: fixed, thx for reminding

File: paddle/fluid/operators/conv_fusion_op.cu.cc (hunk 2 above)
Comment (Shixiaowei02) [reply paddle/fluid/operators/conv_fusion_op.cu.cc]: fixed, thx for reminding

File: paddle/fluid/inference/analysis/ir_passes/subgraph_util.cc [hunk 3]
```diff
@@ -60,6 +60,7 @@ void RenameAndGetOutputs(
     std::set<std::string> *output_names_with_id,
     std::set<std::string> *output_names,
     std::unordered_map<std::string, std::string> *output_name_map,
+    std::unordered_map<std::string, framework::ir::Node *> *graph_var_map,
```
Comment (NHZlX) [inline paddle/fluid/inference/analysis/ir_passes/subgraph_util.cc]: google code style:

change
 ```std::unordered_map<std::string, framework::ir::Node *> *graph_var_map ``` 

to

 ```const std::unordered_map<std::string, framework::ir::Node *>& graph_var_map```

**Review by NHZlX — DISMISSED** [review]
LGTM

**Review by NHZlX — APPROVED** [review]
lgtm

**Pull request merged**